docker-compose.yml

```diff
@@ -1,7 +1,8 @@
 version: '2'
 services:
   autopush:
-    image: bbangert/autopush
+    # image: bbangert/autopush
```

Comment:
Just delete this?

Reply:
I was actually debating if we wanted to make this use the image we generate when we tag  a release, rather than doing a local build.